tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -132,95 +81,9 @@ def test_post_sms_notification_with_persistance_in_celery_returns_201(
     assert mocked.called
 
 
-def test_post_sms_notification_uses_inbound_number_as_sender(notify_api, client, notify_db_session, mocker):
-    notify_api.config["FF_NOTIFICATION_CELERY_PERSISTENCE"] = False
-    service = create_service_with_inbound_number(inbound_number="1")
-    template = create_template(service=service, content="Hello (( Name))\nYour thing is due soon")
-    mocked = mocker.patch("app.celery.provider_tasks.deliver_sms.apply_async")
-    data = {
-        "phone_number": "+16502532222",
-        "template_id": str(template.id),
-        "personalisation": {" Name": "Jo"},
-    }
-    auth_header = create_authorization_header(service_id=service.id)
-
-    response = client.post(
-        path="/v2/notifications/sms",
-        data=json.dumps(data),
-        headers=[("Content-Type", "application/json"), auth_header],
-    )
-    assert response.status_code == 201
-    resp_json = json.loads(response.get_data(as_text=True))
-    assert validate(resp_json, post_sms_response) == resp_json
-    notifications = Notification.query.all()
-    assert len(notifications) == 1
-    notification_id = notifications[0].id
-    assert resp_json["id"] == str(notification_id)
-    assert resp_json["content"]["from_number"] == "1"
-    assert notifications[0].reply_to_text == "1"
-    mocked.assert_called_once_with([str(notification_id)], queue="send-sms-tasks")
-
-
-def test_post_sms_notification_uses_inbound_number_reply_to_as_sender(notify_api, client, notify_db_session, mocker):
```

Comment:
We might want to change this test instead of axing it, by mocking the celery database-task and checking what it receives maybe?